models/oauth2.go

```diff
@@ -40,7 +40,7 @@ var OAuth2Providers = map[string]OAuth2Provider{
 			ProfileURL: oauth2.GetDefaultProfileURL("gitlab"),
 		},
 	},
-	"gplus":         {Name: "gplus", DisplayName: "Google+", Image: "/img/auth/google_plus.png"},
+	"gplus":         {Name: "gplus", DisplayName: "Google+", Image: "/img/auth/google.png"},
```

Comment:
```suggestion
	"gplus":         {Name: "gplus", DisplayName: "Google", Image: "/img/auth/google.png"},
```
We can maybe rename it ?

Reply:
Ah, I missed that rename. Thanks for catching :)